sklearn/metrics/_plot/det_curve.py

```diff
@@ -113,6 +115,11 @@ def from_estimator(
         sample_weight : array-like of shape (n_samples,), default=None
             Sample weights.
 
+        drop_intermediate : bool, default=True
+            Whether to drop thresholds where true positives (tp) do not change
+            from the previous or subsequent threshold. All points with the same
+            tp value have the same `fnr` and thus same y coordinate.
```

Comment:
@ArturoAmorQ could you please open a follow-up PR to add the missing `..versionadded` markers for the new public parameters?